cmd/openshift/openshift.go

```diff
@@ -13,7 +13,9 @@ func main() {
 	defer serviceability.BehaviorOnPanic(os.Getenv("OPENSHIFT_ON_PANIC"))()
 	defer serviceability.Profile(os.Getenv("OPENSHIFT_PROFILE")).Stop()
 
-	runtime.GOMAXPROCS(runtime.NumCPU())
+	if len(os.Getenv("GOMAXPROCS")) == 0 {
+		runtime.GOMAXPROCS(runtime.NumCPU())
```

Comment:
FWIW defaulting to all cores is already on tip of Go and will ship with 1.5
https://github.com/golang/go/commit/3ffcbb633e95b17389ce5ce2f6b5dbcbbdf88828

+1 for making this configurable. We should doc it somewhere.